### DISCUSSION TITLE
read only access ?

### DISCUSSION THREAD

**Discussion by gurusonwheels** [nlef/moonraker-telegram-bot#110] [Q&A] [answered]
are there a way to allow my wife to monitor my prints on the telegram bot 
but not have any control access at all ?

i would like her to be able to see whats happening on the printer 
but not issue commands .

**Comment by nlef** [comment, marked as answer]
@gurusonwheels 
Sorry for the long delay in reply.
You can use the functionality of sending notifications to groups/channels from the section [progress_notification](https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config#progress_notification)

**Reply by Meta-Cake** [reply]
So there is no way to make a group where anyone can ping the bot with /status or /video to see what the printer is doing?
